src/components/date-picker/calendar.tsx

```diff
@@ -186,6 +228,18 @@ export class Calendar extends React.Component<CalendarProps, {}> {
         }
     }
 
+    private selectMonth: React.EventHandler<React.SyntheticEvent<Element>> = event => {
+        event.preventDefault();
+        const eventTarget = event.target as HTMLSpanElement;
```

Comment:
Since you are using eventTarget only once, I think it will be better to just use
(event.target as HTMLSpanElement).textContent!